utilities/ltftUtilities.ts

```diff
@@ -376,18 +376,18 @@ export function getStatusReasonLabel(
   if (status === "UNSUBMITTED" && statusReason) {
     return (
       ACTION_REASONS.UNSUBMIT.find(reason => reason.value === statusReason)
-        ?.label ?? "Other reason"
+        ?.label ?? statusReason
     );
   }
 
   if (status === "WITHDRAWN" && statusReason) {
     return (
       ACTION_REASONS.WITHDRAW.find(reason => reason.value === statusReason)
-        ?.label ?? "Other reason"
+        ?.label ?? statusReason
     );
   }
 
-  return "";
+  return statusReason || "";
```

Comment:
This is to fix the value mismatch when LTFT is unsubmitted from TIS Admins-UI with reason `Proposed start date` being shown as `other reason`. Will show the actual value of UNSUBMITTED and WITHDRAWN reason instead of "other".